sprint-reports/sprint-1.md

```diff
@@ -0,0 +1,113 @@
+# Sprint N°1 [01/01/2024 - 04/02/2024]
+
+_Initial configuration so that everyone can start the project in a clean environment._
+
+## Positive reviews
+
+- Review 1
+
+  - Description: A dynamic, motivated team that does its utmost to give the best of itself.
+  - Commitment:
+    - Who: Barnabé
```

Comment:
I think it would be clearer to say who said it instead of just saying "Who". I suggest to replace it by "Said by"

Reply:
```suggestion
    - Said by: Barnabé
```